Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -106,9 +79,9 @@ public struct ConvertAction: AsyncAction {
         emitDigest: Bool,
         currentPlatforms: [String : PlatformVersion]?,
         buildIndex: Bool = false,
-        workspace: DocumentationWorkspace = DocumentationWorkspace(),
-        context: DocumentationContext? = nil,
-        dataProvider: DocumentationWorkspaceDataProvider? = nil,
+        workspace _: DocumentationWorkspace = DocumentationWorkspace(),
```

Comment:
Binding the parameter to `_` is a way of explicitly marking it as unused because nothing can access its value. Callers can still pass a value but the received doesn't (and can't) access it. I used it here to verify for myself that nothing was using the workspace. 

The reason for not removing the parameter completely is that there were still tests that used it and I didn't want to make this PR bigger than it already was. Instead I removed this parameter [here](https://github.com/swiftlang/swift-docc/pull/1059/files#diff-be716f893a903f4ca669b5d3a4306bb39e745fffbcdea448bcb3640831bc25b5L109) in #1059